src/gui/viewer.cpp

```diff
@@ -248,6 +244,12 @@ void ViewerMainWindow::setup_menubar() {
 		{
 			this->reload_file();
 		});
+	fileMenu->addAction(
+		style()->standardIcon(QStyle::SP_ArrowUp), "Import File",
```

Comment:
For the icon, we can use `QIcon::fromTheme("document-import", style()->standardIcon(QStyle::SP_ArrowUp))`.

`document-import` with most custom Qt themes (such as KDE breeze) will resolve to a more appropriate import file icon. `SP_ArrowUp` can be used as a good fallback here.